components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -396,12 +399,47 @@ public class LegacyJobOrchestrator : IJobOrchestrator, IPublisherConfigServices,
             _agentConfig.TriggerConfigUpdate(this, new EventArgs());
         }
 
+        private static void AdjustResponse(ref List<string> statusResponse) {
```

Comment:
I'm not sure if this adjustment is the right approach here. For the response payloads that we expect to exceed the DM response capacity we have to implement a continuation token mechanism. That mechanism should be part of of the method router though, not in the orchestrator.